themes/classic/assets/js/theme.js

```diff
@@ -9191,7 +9191,7 @@
 /***/ function(module, exports, __webpack_require__) {
 
 	var __WEBPACK_AMD_DEFINE_FACTORY__, __WEBPACK_AMD_DEFINE_ARRAY__, __WEBPACK_AMD_DEFINE_RESULT__;/*!
-	 * Datepicker for Bootstrap v1.6.0 (https://github.com/eternicode/bootstrap-datepicker)
+	 * Datepicker for Bootstrap v1.5.1 (https://github.com/eternicode/bootstrap-datepicker)
```

Comment:
Well. As we use "^x.y.z" syntax in package.json file each person potentially has a different set of external components. So we need to launch "npm update" from time to time and make sure we have latest version before compiling assets for PR.